From 136622acc965ac51d43148416210f1a615b58119 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Sat, 14 Nov 2020 16:49:22 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: Increase PPLL to 200MHz

ppll is for combophy refclk to lock, and it should support
25M/100M. So we need to increase it to 200MHz.

Change-Id: I224a221296ed074a60354d65e7c678de4f7f7120
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 90c3edac565f..0cde1070bc32 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -567,7 +567,7 @@
 			<&cru ACLK_PERIMID>, <&cru HCLK_PERIMID>,
 			<&cru PLL_NPLL>;
 		assigned-clock-rates =
-			<32768>, <100000000>,
+			<32768>, <200000000>,
 			<100000000>, <1000000000>,
 			<1188000000>, <600000000>,
 			<150000000>, <100000000>,
-- 
2.35.3

